pom.xml

```
@@ -121,6 +121,12 @@
             <version>2.0.1.Final</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.gitlab4j</groupId>
+            <artifactId>gitlab4j-api</artifactId>
+            <version>4.18.0</version>
```

Suggest using the latest Stable version of this package.
```suggestion
            <version>4.19.0</version>
```